File: Prod.RutaDigital.Entidades/Respuesta/Respuesta.cs
namespace Prod.RutaDigital.Entidades;

public class Respuesta : Auditoria
{
    public int id_respuesta { get; set; }
    public int id_evaluacion { get; set; }
    public int id_modulo { get; set; }
    public decimal peso_modulo { get; set; }
    public int id_pregunta { get; set; }
    public int id_alternativa { get; set; }
    public decimal peso_alt { get; set; }
    public bool respuesta { get; set; }
}

File: Prod.RutaDigital.Presentacion.Angular/ClientApp/src/app/autodiagnostico/interfaces/respuesta.ts
export interface Respuesta {
  id_respuesta?: number;
  id_evaluacion?: number;
  id_modulo?: number;
  peso_modulo?: number;
  id_pregunta?: number;
  id_alternativa?: number;
  peso_alt?: number;
  respuesta?: boolean;

  orden_modulo?: number;
  nombre_modulo?: string;
  descripcion_modulo?: string;
  imagen_modulo?: string;

  orden_pregunta?: number;
  descripcion_pregunta?: string;

  orden_alternativa?: number;
  descripcion_alternativa?: string;
}
